### ISSUE TITLE
Windows: ValueError: need more than 0 values to unpack

### ISSUE THREAD

**Issue by hugovk** [tldr-pages/tldr-python-client#9]
Python tldr isn't working for me under Windows 7 native command prompt:

```
C:\>python --version
Python 2.7.8

C:\>pip install tldr
Collecting tldr
  Downloading tldr-0.1.1.tar.gz
Requirement already satisfied (use --upgrade to upgrade): six in c:\python27\lib\site-packages (from tldr)
Requirement already satisfied (use --upgrade to upgrade): termcolor in c:\python27\lib\site-packages (from tldr)
Installing collected packages: tldr
  Running setup.py install for tldr
    Installing tldr-script.py script to C:\Python27\Scripts
    Installing tldr.exe script to C:\Python27\Scripts
Successfully installed tldr-0.1.1

C:\>pip freeze | grep tldr
tldr==0.1.1

C:\>tldr tldr
Traceback (most recent call last):
  File "C:\Python27\Scripts\tldr-script.py", line 9, in <module>
    load_entry_point('tldr==0.1.1', 'console_scripts', 'tldr')()
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 521, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 2632, in load_entry_point
    return ep.load()
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 2312, in load
    return self.resolve()
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 2318, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "C:\Python27\Scripts\tldr.py", line 13, in <module>
    rows, columns = map(int, os.popen('stty size', 'r').read().split())
ValueError: need more than 0 values to unpack
```

Similarly under Git Bash on Windows 7:
```
$ python --version
Python 2.7.8

$ pip install tldr
Requirement already satisfied (use --upgrade to upgrade): tldr in c:\python27\lib\site-packages
Requirement already satisfied (use --upgrade to upgrade): six in c:\python27\lib\site-packages (from tldr)
Requirement already satisfied (use --upgrade to upgrade): termcolor in c:\python27\lib\site-packages (from tldr)

$ pip freeze | grep tldr
tldr==0.1.1

$ tldr tldr
Traceback (most recent call last):
  File "c:\Python27\Scripts\tldr-script.py", line 9, in <module>
    load_entry_point('tldr==0.1.1', 'console_scripts', 'tldr')()
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 521, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 2632, in load_entry_point
    return ep.load()
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 2312, in load
    return self.resolve()
  File "C:\Python27\lib\site-packages\pkg_resources\__init__.py", line 2318, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "c:\Python27\Scripts\tldr.py", line 13, in <module>
    rows, columns = map(int, os.popen('stty size', 'r').read().split())
ValueError: need more than 0 values to unpack
```

It's working under Linux (Ubuntu virtual machine):
```
$ python --version
Python 2.7.3
$ pip install tldr
Requirement already satisfied (use --upgrade to upgrade): tldr in /usr/local/lib/python2.7/dist-packages
Requirement already satisfied (use --upgrade to upgrade): six in /usr/local/lib/python2.7/dist-packages (from tldr)
Requirement already satisfied (use --upgrade to upgrade): termcolor in /usr/local/lib/python2.7/dist-packages (from tldr
)
Cleaning up...
$ pip freeze | grep tldr
Warning: cannot find svn location for distribute==0.6.24dev-r0
tldr==0.1.1
$ tldr tldr
# tldr



  Simplified man pages



- get typical usages of a command (hint: this is how you got here!)



  tldr {{command}}
```

**Comment by Anmol-Singh-Jaggi** [comment]
It won't work on Windows as `stty` command is not available on Windows.

**Comment by hugovk** [comment]
Aha, thanks. `stty` is only used to get the number of columns in the terminal. 

It would be good to default to, say, 80 for Windows with a `try`/`except`, or handle it properly with perhaps something like https://gist.github.com/jtriley/1108174

**Comment by Anmol-Singh-Jaggi** [comment]
Good idea! I'll implement and push it when I get the time. By the way, there is a built-in function `shutil.get_terminal_size()` in Python3.

**Comment by Anmol-Singh-Jaggi** [comment]
I dont't think this module can easily be ported to Windows. Apart from this terminal size issue, there are more problems - termcolor and six do not work on Windows.
Although, I think things will get easier if we translate this to Python3.

**Comment by Anmol-Singh-Jaggi** [comment]
[Done!](https://github.com/tldr-pages/tldr-python-client/pull/18)

**Comment by hugovk** [comment]
Thanks, I'll test it out next week!

**Comment by hugovk** [comment]
Thanks, it works on Windows now!
